apps/envited-x-data-space/app/api/tokens/[scoped-identifier]/route.ts

```diff
@@ -0,0 +1,36 @@
+import { NextResponse } from 'next/server'
+import { isEmpty, isNil, prop } from 'ramda'
+
+import { getServerSession } from '../../../../common/auth'
+import { ERRORS, ERROR_CODES } from '../../../../common/constants/errors'
+import { db } from '../../../../common/database/queries'
+
+export async function GET(request: Request, { params }: { params: { 'scoped-identifier': string } }) {
```

Comment:
Given that a scoped identifier is the last part of *any* uri formatted identifier the use of the word scopedIdentifier as the parameter here but also in the fetchTokenByScopedIdentifier function name. What you're *actually* doing is finding a token its domainMetadataId (I think)